examples/errorprone/Hello.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.examples.errorprone;
+
+// import com.google.startupos.proto_vs_json.Protos;
```

Comment:
Leftover: `// import com.google.startupos.proto_vs_json.Protos;`

Reply:
@oferb it was intentional 🙂 
You'll need it if you uncomment [this](https://github.com/google/startup-os/blob/error-prone-bazel-check/examples/errorprone/Hello.java#L29) example

Reply:
Ohh, so maybe leave it uncommented?

Reply:
@oferb then IntelliJ IDEA is complaining and there's no way to suppress it – but we can do it

Reply:
@oferb I left a comment describing why we need it in 4d17c6f